include/cudaq/Optimizer/Transforms/Passes.td

```diff
@@ -327,7 +327,7 @@ def QuakeAddMetadata : Pass<"quake-add-metadata", "mlir::func::FuncOp"> {
   let constructor = "cudaq::opt::createQuakeAddMetadata()";
 }
 
-def QuakeObserveAnsatz : Pass<"quake-observe-ansatz", "mlir::func::FuncOp"> {
+def QuakeObserveAnsatz : Pass<"observe-ansatz", "mlir::func::FuncOp"> {
```

Comment:
If we're changing the pass string name, I'd imagine we'd want to change the pass name (i.e. QuakeObserveAnsatz --> ObserveAnsatz) in order to keep things consistent.

Reply:
We can do that later. The name in the code is not a customer facing issue.